src/components/organisms/NavigatorPane/NavigatorPane.tsx

```diff
@@ -18,16 +19,27 @@ import SectionRenderer from './SectionRenderer';
 import WarningsAndErrorsDisplay from './WarningsAndErrorsDisplay';
 import * as S from './NavigatorPane.styled';
 
-const NavPane = () => {
+const NavPane: React.FC = () => {
   const dispatch = useAppDispatch();
   const {windowSize} = useContext(AppContext);
   const windowHeight = windowSize.height;
   const navigatorHeight = windowHeight - NAVIGATOR_HEIGHT_OFFSET;
 
   const fileMap = useAppSelector(state => state.main.fileMap);
+  // TODO: remove this any
+  const resourceFilters: any = useAppSelector(state => state.main.resourceFilter);
+
   const isInClusterMode = useSelector(isInClusterModeSelector);
   const isInPreviewMode = useSelector(isInPreviewModeSelector);
 
+  const appliedFilters = useMemo(() => {
+    return Object.keys(resourceFilters)
+      .map(filterName => {
```

Comment:
You could use `Object.entries()` instead of `.keys` and then `.map`

Reply:
Thanks for the advice!